utils/README.md

```diff
@@ -0,0 +1,78 @@
+# setup
+
+For the Python part, we use [Poetry] for package management.
+
+- On the Debian family of OSes, install it so:
+
+  ```
+  apt install python3-poetry
+  ```
+
+- Proceed by navigating to a directory containing pyproject.toml,
+  then use `poetry` to install some needed packages:
+
+  ```
+  cd $enclave/app
```

Comment:
`utils/enclave` directory does not have any `pyproject.toml`

Reply:
yeah, it's not needed... Python bindings are only needed for the app dir

Reply:
then shall we remove "$enclave" since there's no python bindings in there?

Reply:
oh, indeed

Reply:
this fix doesn't remove "enclave" from the cd instruction

Reply:
https://github.com/ntls-io/ntls-sgx-service/pull/5/commits/a1437eed419f4d5d862d4bcd736fe0764cf0b951